.github/workflows/deploy_feature_branch.yml

```diff
@@ -1,4 +1,4 @@
-name: "Reusable - Deploy Feature Branch to Dev"
+name: "POC - Deploy Feature Branch to Dev"
```

Comment:
discard all other changes to this file since you aren't using them?

Reply:
I discarded some of the changes, but I kept the ones related to oidc and slack notification for consistency